amqp-impl/src/test/java/io/streamnative/pulsar/handlers/amqp/test/AmqpChannelMethodTest.java

```diff
@@ -103,11 +103,11 @@ public void testAccessRequest() {
         Assert.assertEquals(accessRequestOkBody.getTicket(), 0);
     }
 
-    @Test
+//    @Test
```

Comment:
What's going on here?  Is this test not passing?

If it's not passing, mark it with the `@Ignore("description of why it's ignored")`, instead of commenting it out.

Reply:
Oh, it is useful, I will remove the "//".